Dominio/Modelos/Raca.cs

```diff
@@ -9,12 +9,7 @@ public class Raca
         public string? HabilidadeRacial { get; set; }
 
         public Raca() { }
```

Comment:
se o construtor for vazio, nao precisa colocar ele. por padrao, toda classe ja tem um construtor vazio, desde que nao haja outro construtor com parametro na sua classe. se esse fosse o caso, ai sim, vc poderia colocar esse construtor

Reply:
removido